### DISCUSSION TITLE
ArgumentError: Could not parse rfc1738 URL

### DISCUSSION THREAD

**Discussion by darkshade9** [getredash/redash#5994] [Q&A] [answered]
Hello,

I've recently transferred my setup over to a new machine, but when I go to bring up the stack (`docker-compose up -d`), I am getting this error from the redash_server_1 instance:

`ArgumentError: Could not parse rfc1738 URL from string '"postgresql://postgres:${POSTGRES_PASSWORD}@postgres/postgres"'`

I did not edit this file when I transferred the entire directory over to the new host.  In fact I ran an md5sum against both the source and destination files, they are identical.  Why does the new one complain while the old one works fine?

`REDASH_DATABASE_URL="postgresql://postgres:${POSTGRES_PASSWORD}@postgres/postgres"`

Thanks

**Comment by justinclift** [comment]
In theory, the `POSTGRES_PASSWORD` value gets filled out on the fly using whatever value is in `/opt/redash/env`.

That error message kind of seems like something is going wrong there.

Hmmm, there's also the two environment variables set in the `.profile` of whichever OS user runs Redash:

```
export COMPOSE_PROJECT_NAME=redash
export COMPOSE_FILE=/opt/redash/docker-compose.yml
```

Not sure if you copied across the users' `.profile` file from their home directory or not.  If not, and you have the old machine around still... then maybe look at that? :smile:

**Comment by darkshade9** [comment, marked as answer]
I found the answer, my docker-compose was a bit older (1.17), upgrading to 1.29 seems to have done the trick.  Something about the way that the older docker-compose was handling the env file wasn't quite right.

**Comment by justinclift** [comment]
Awesome. :smile:
